integration/client_get_config_test.go

```diff
@@ -0,0 +1,94 @@
+package integration
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+	"testing"
+)
+
+type Client struct {
```

Comment:
I wonder could we reuse the ```MobileClientSpec``` in ```client_crudl_test.go``` I am actually a fan of creating a file in a package called ```types.go``` in which are defined the shared types for that package